phpunit.xml.dist

```diff
@@ -57,4 +57,9 @@
 	<testsuite name="backend-mixed">
 		<directory>tests/Backend/Mixed</directory>
 	</testsuite>
+
+	<testsuite name="storage-types-feature">
+		<file>tests/Backend/Workspaces/MetadataFromRedshiftWorkspaceTest.php</file>
+		<file>tests/Backend/Workspaces/MetadataFromSnowflakeWorkspaceTest.php</file>
```

Comment:
This one to the `backend-snowflake-part-1`

Reply:
same, needs the feature enabled, that's why it's in a different suite.  once feature is removed they will be moved to those suites like you suggest